index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+  </head>
+
+  <body>
+    <div class="template5-main-mobile">
+      <div class="app-bar">
+        <div class="time"><img src="Time.png" alt="time" /></div>
+        <img class="name" src="name.png" alt="name " />
+        <div class="Cellular_Connection-path1"></div>
+        <div class="Cellular_Connection-path2"></div>
+        <div class="Cellular_Connection-path3"></div>
+        <div class="Cellular_Connection-path4"></div>
+        <div class="battery-border"></div>
+        <div class="battery-capacity"></div>
+        <div class="battery-cap"></div>
+        <div class="union">
+          <div class="vector1"></div>
+          <div class="vector2"></div>
+          <div class="vector3"></div>
+        </div>
```

Comment:
- [ ] I suggest using [Font Awesome](https://fontawesome.com/) hamburger icons or images to display hamburgers. This hamburger icon should be inside the `nav` tag.
- **Requirement** The hamburger button is on the right, and has a `nav` tag.